src/tabs/structure.tsx

```diff
@@ -65,8 +65,18 @@ const Side = styled("div")({
 });
 Side.displayName = "Side";
 
-const Center = styled("div")({
+const StartSide = styled("div")({
```

Comment:
This component is not used anywhere.

Reply:
not show it used i think because of merge commits :) should be good now